### ISSUE TITLE
Classic maxUnitCount increments when players log in or out

### ISSUE THREAD

**Issue by Wizeowel** [WeakAuras/WeakAuras2#2118]
## Describe the bug
I have a weakaura on my priest that should pop up when someone in my party doesn't have fort or spirit buff. Trigger 1 checks for Aura|Group|contains 'Fortitude'|Unit Count < 100%. In the display I show text %1.unitCount/%1.maxUnitCount.

Since 2.17 when someone from the party logs out maxUnitCount increases by double, and when they log in again it goes 3x higher again. 

E.g. in a party of 2, where one person has fortitude, it would normally show the fort icon and in text 1/2. If other player logs out it goes to 1/4. Then when they log in again it goes to 1/10!

Not only is this visually incorrect, but it also means the trigger becomes active when it shouldn't be. If 2 people have Fortitude that is Unit Count = 100% and the trigger is not active, which is correct. If one logs out then Unit Count is suddenly 1/4 and the trigger becomes active. When they log in again, even though everybody has the buff, the trigger is active because it sees 2/10 and therefore Unit Count < 100%.

**Do you have an error log of what happened?**
No error log.

### To Reproduce
Display text :  %1.unitCount/%1.maxUnitCount
Trigger 1 : Aura|Group|contains 'Fortitude'|Unit Count < 100%

**Did you try having WeakAuras as the only enabled addon and everything else (especially something like ElvUI) disabled?**
Yes

**Which version of WeakAuras are you using?**
2.17.1

**Are you on World of Warcraft Classic or Retail?**
- [x] Classic

**Was it working in a previous version? If yes, which was the last good one?**
Yes, 2.16.x

### Additional Info
!TAv3YjUnu4NMD2n3qIHSDtA70zsiqw2bmSiNMDNoXGWw2wBmsELKHqUWp79CKSbdz60ErNHX4ZV678RCOxyqizDijg)bVslnzs10cdxk0adtiz30KenZe6FbiwebINj5cqq)b(bdMhsIKY8y5wbzlVGfE7bgdItT00Og3r0gQY48lxWRFlbEvNzFhOmkEAktHQ7BHxnd8vUHT2NUMbs62T7hr8QOcGrT2Kju1ZvlLjvlnzSQLpYZJd97gsUNNyoLpCwRPMOmMMKj3ovaOZ()eUwZfPHKsnBrjIsigsvYYIfrYsmY9U4I3bjm2QYKKGDqqt(8GXZg(WyaSwsewDpYOPfmf1ivHKFhCSGTHHocs3sKeodNYagkxzfsaFZFjKSO)nKGfKGBMh4q0(CWTw67BbSBpy(mfZAoz2GXaSAYrW5Oly55JIRPQbYiWPvlh8ZsErbl2P)mQXWuIf47HKHsLHBkJz2t1)GC8uJH4fRXqQ5yBL7dBOih)H3)o97pdpwaT7RQ2QEmx)Jsb0OSbtFIDqxuP2ixh4uBSmLhboXQIu8HARpR6lvlRwQyMsLOAzd7)Y7jOyRGIDnD3NoRAjveFGtVN(TQVWeXUUZnS6ZzImgBUC9NyeqZ)tOxedn)UxfsEMXkUbYGrM5yahUcYNS8KtMiIzAQbtjmuHnnoONT8mNLwpnu3EZEXSqNrHjMV1AuZYgFesENxhSfPpwJphiwtF5Hg6JSVVmxQCogCb03Jp7zFEzDu50(aKV5HGP1CXMryGGh9ip2KbISnNwrqRel2X2)xUSzpW3pfU)akz8KDhsekPbscb2OWFQ)GM5din4cnRzrTGTNf2Ewy7DcStKiIhQ4VwT8RL0yibtRwgCCo8nGARuf)OIcJwpU)nRGnCnFvUDoYf)TxUnY3FW8f3oniy6K5JU)ZbTWaH)k0KCDtA4nvnuh3AHPpemEee2gmQaxKMl3ouX(zjteTdmOdUdBFkbL60PTlrAN3MbvHChJXmrkwm8Qv47Nyq))9ukU7yVMRCMz30HNxgp6zbtRTlvroKiAoBp1yoim0)kh1TqELH(WojNX4Pzas6blaZLu71kW6u72n8YMYCd3MlSCGdur5XW)MAqTaxoAreoN12K9A04oqNCq58wRz(dNgw2OZDVqGIvuov7UdcmhIfYDZFy0DTaKJUgiASgFYzBAQ4dvWko7dOOKt3Xu1ek7STRAXJW9H2NaoEPvbc21vawncf5717JxF1NoCPjoSChxtHoZy7cD2AmDq86aDlhbGzufU7EfARiHNwFzkkcUZeMnSLRxLY1U7UXXzqzJmA)gjpVExaJZq(NuFRx1Y9l6HrlM(4y2fzK(ZhmWheKxK5oKTUvdyj))WWCjEEpQ)6vxFxIVAIg7pqiz3wwQGYvMXuO)1ZpFlnv2Hlp)Yj3F93gMoZ)8E2OnM36duAYC2p1qWx7UjQ1hBupHjKc2HRQwup2Xq3bqGr1yAREUQNJrWbdTFTaxCWDfkgwqpvV98lkZHi6T(9FaaT)oO)NaSRroK0TJ3N6avPnq()YUyji8Vd

**Comment by Wizeowel** [comment]
Also noticed another idiosyncrasy regarding players logging out/in and joining/leaving the party since 2.17. 

For example in 2.16 I used Status|Unit Characteristics|Specific Unit to provide an 'always on' trigger for when a particular person is in my group. This trigger works now only for the most recent person added to the group, and I must open /wa and close it again *each time the group composition changes* for the trigger(s) to activate! I tried a work around with Status|Health|Specific Unit|Health>=0 but this has the same issue.
